src/test/java/org/opensearch/security/securityconf/impl/v7/ConfigV7Test.java

```diff
@@ -13,10 +13,11 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.google.common.collect.ImmutableList;
-import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
```

Comment:
Pretty sure there is a CI failure because `*` imports shouldn't be allowed, please expand this out.

Reply:
`gradle checkstyleMain checkstyleTest` and `gradle spotlessCheck spotlessApply` commends help to solve such problems :-)

Reply:
Thank you ! :)